db/models/getArticles.model.js

```diff
@@ -1,11 +1,45 @@
 const db = require("../connection");
 
-exports.findArticles = () => {
-  return db
-    .query(
-      `SELECT 
+exports.findArticles = (topic) => {
+  if (topic) {
+    return db
+      .query(
+        `SELECT 
     a.article_id, 
-    a.title, topic, 
+    a.title, 
+    a.topic, 
+    a.author, 
+    a.created_at, 
+    a.votes, 
+    a.article_img_url, 
+    COUNT(c.comment_id) AS comment_count 
+    FROM articles a
+    LEFT JOIN comments c ON a.article_id = c.article_id
+    WHERE a.topic = $1
+    GROUP BY a.article_id, a.title, a.topic, a.author, a.created_at, a.votes, a.article_img_url
+    ORDER BY a.created_at DESC`,
+        [topic]
+      )
+      .then(({ rows }) => {
+        const articles = rows[0];
+
+        if (!articles) {
```

Comment:
Do we want to reject this if there are no articles? What if it's an existing topic with no articles yet?